test/integration/query-interface/schemas.test.ts

```diff
@@ -3,12 +3,14 @@ import { spy } from 'sinon';
 import type { CreateSchemaQueryOptions } from '@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/abstract/query-generator';
 import { sequelize } from '../support';
 
-const queryInterface = sequelize.queryInterface;
-
+const { queryInterface, dialect } = sequelize;
 const testSchema = 'testSchema';
 
-describe('QueryInterface#createSchema', async () => {
-  if (!sequelize.dialect.supports.schemas) {
+// MySQL and MariaDB view databases and schemas as identical. Other databases consider them separate entities.
+const dialectsWithEqualDBsSchemas = ['mysql', 'mariadb'];
+
+describe('QueryInterface#{create,drop,dropAll,showAll}Schema', () => {
+  if (!dialect.supports.schemas) {
```

Comment:
this test file should be aligned with `create-drop-list-database.test.ts` imo. Dialects that don't support schemas should be tested too, but the methods should throw